src/commands/bitbucket-server.ts

```diff
@@ -1,13 +1,15 @@
-import { Flags } from '@oclif/core';
+import { Command, Flags } from '@oclif/core';
 import { vcsServerFlags } from '../common/flags';
 import { MAX_REQUESTS_PER_SECOND_VAR } from '../common/throttled-vcs-api-manager';
-import { HelpGroup, SourceType, VcsSourceInfo } from '../common/types';
+import { HelpGroup, SourceType } from '../common/types';
 import { deleteFlagKey, getServerUrl, init } from '../common/utils';
 import { BitbucketServerApiManager } from '../vcs/bitbucketServer/bitbucket-server-api-manager';
 import { BitbucketServerRunner } from '../vcs/bitbucketServer/bitbucket-server-runner';
 import Bitbucket from './bitbucket';
+import { BitbucketServerVcsSourceInfo } from '../vcs/bitbucketServer/bitbucket-server-types';
 
-export default class BitbucketServer extends Bitbucket {
+// do not extend Bitbucket command, because we do not want to inherit all of the flags (workspaces is not relevant)
```

Comment:
oh well 🤷🏽

Reply:
I already had created a mechanism to pick and choose flags to reuse, and I never noticed that it inherited the flags, so this is fine. Before this change, the line that pulled in the `Bitbucket` flags was actually useless, because that already was happening.